lib/sparrow/pools_warden.ex

```diff
@@ -132,8 +166,19 @@ defmodule Sparrow.PoolsWarden do
         }, pool_name=#{inspect(pool_name)}, pool_tags=#{inspect(tags)}"
       end)
 
-    {:reply, pool_name,
-     Map.merge(state, %{pid => [pool_type, pool_name, tags]})}
+    new_state = Map.merge(state, %{pid => [pool_type, pool_name, tags]})
+
+    :telemetry.execute(
+      [:sparrow, :pools_warden, :add_pool],
+      %{},
+      %{
+        pool_name: pool_name,
+        pool_type: pool_type,
+        tags: tags
```

Comment:
```suggestion
        pool_tags: tags
```

Let's make all labels that mean the same thing, actually the same.